lib/index.js

```diff
@@ -66,19 +66,19 @@ Typpy.get = function (input, str) {
         return str ? "string" : String;
     }
 
-    if (null === input) {
+    else if (null === input) {
```

Comment:
We don't really need `else` here due to the `return` statements.

Thanks!

Reply:
Although not required by any means, because of the return, it's still a good practice for readability and clean code. 

If you'd still like me to remove them I will, but seems better to have them than not. :+1:

Reply:
I think it's better to remove it. In other cases it helps, but here I rather think it is clearer since we have many returns on the same level.